var/spack/repos/builtin/packages/rocm-cmake/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class RocmCmake(CMakePackage):
+    """ROCM cmake modules provides cmake modules for common build tasks
+       needed for the ROCM software stack"""
+
+    homepage = "https://github.com/RadeonOpenCompute/rocm-cmake"
+    url      = "https://github.com/RadeonOpenCompute/rocm-cmake/archive/rocm-3.5.0.tar.gz"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    version('3.5.0', sha256='5fc09e168879823160f5fdf4fd1ace2702d36545bf733e8005ed4ca18c3e910f')
+
+    variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
+
+    depends_on('cmake@3.5.2', type='build')
```

Comment:
https://github.com/RadeonOpenCompute/rocm-cmake/blob/master/CMakeLists.txt#L5 here it says "cmake_minimum_required (VERSION 2.8) # TODO: Should probably be 3.0", which is a lowerbound, maybe just replace this with `depends_on('cmake', type='build')`?

Reply:
I find this little tricky.. version inside cmakelists.txt files mention cmake_minimum_required( Versions) appears very different for each of the rocm components. I am seeing conflicts due to this when spack is trying to concretize spec Already requested spack on it

Reply:
I mentioned this above, but it should be:
```suggestion
    depends_on('cmake@2.8:', type='build')
```
The colon at the end is very important, it means that 2.8 is the minimum required version, but there is no maximum required version, it's an open range.